scripts/src/owners/checkuser.py

```diff
@@ -0,0 +1,68 @@
+import re
+import argparse
+import requests
+import os
+import sys
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+def verify_user(username):
+    print(f"[INFO] Verify user. {username}")
+    owners_path = "OWNERS"
+    if not os.path.exists(owners_path):
+        print(f"[ERROR] {owners_path} file does not exist.")
+    else:
+        data = open(owners_path).read()
+        out = yaml.load(data, Loader=Loader)
+        if username in out["approvers"]:
+            print(f"[INFO] {username} authorized")
+            return True
+        else:
+            print(f"[ERROR] {username} not auhtorized")
+    return False
+
+def check_for_restricted_file(api_url):
+    files_api_url = f'{api_url}/files'
+    headers = {'Accept': 'application/vnd.github.v3+json'}
+    pattern_owners = re.compile(r"OWNERS")
+    pattern_versionfile = re.compile(r"cmd/release/release_info.json")
+    pattern_thisfile = re.compile(r"scripts/src/owners/checkuser.py")
```

Comment:
Do we need the regex for `pattern_owners`, `pattern_versionfile`, and `pattern_thisfile`? They can all be string constants.

Reply:
Done